ui/src/badge_table_cell_type.tsx

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020 H2O.ai, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { S, U } from 'h2o-wave'
+import React from 'react'
+import { stylesheet } from 'typestyle'
+import { clas, cssVar, getContrast, padding } from './theme'
+
+
+/** 
+ * Creates a collection of chips, usually used for rendering state values.
+ * Note: In case of multiple tags per row, make sure the row values are
+ * separated by "," within a single cell string.
+ * E.g. ui.table_row(name='...', cells=['cell1', 'BADGE1,BADGE2']).
```

Comment:
Describe how the ,-separated values are used to render the badge. Do values correspond to the `name`?